code/modules/shuttle/super_cruise/orbital_map_components/orbital_objects/phobos.dm

```diff
@@ -0,0 +1,11 @@
+//Centcom Z-Level.
+//Syndicate infiltrator level.
+//TODO: Normal ships arriving here will crash on a snowy z-level.
```

Comment:
neat

Reply:
uhhhhh idk if its worth making an extra z-level that wont be used outside of this